cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1374,22 +1387,99 @@ private KafkaVersionChange getKafkaVersionChange(StatefulSet kafkaSts) {
+
+                                return Future.failedFuture(new IllegalArgumentException(scaleErrorMessage));
+
+                            } else if (desiredReplicas > currentReplicas) {
+                                // If this is a scale up of a 3.4.x cluster we want to scale in steps of one at a time
+                                // automatically.
+                                zkCluster.setReplicas(currentReplicas + 1);
+                                zkScalingUp = true;
+                            } else if (desiredReplicas < currentReplicas) {
+                                zkScalingDown = true;
+                            }
+
+                            ConditionBuilder zkManualScalingCondition = null;
+
+                            if ((zkScalingUp || zkScalingDown) && isZK35x) {
+                                // This is a scale up or down of a ZK 3.5.x cluster
+
+                                String scaleMessage = String.format(
+                                        "Scaling Zookeeper 3.5.x cluster: Rolling updates will be skipped and the manual " +
+                                        "update process defined in the documentation should be followed. When finished set " +
+                                         "%s annotation on the Zookeeper StatefulSet to false.",
+                                        Annotations.ANNO_STRIMZI_IO_MANUAL_ZK_SCALE);
+
+                                log.info(scaleMessage);
 
+                                zkManualScalingCondition = new ConditionBuilder()
+                                        .withLastTransitionTime(new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ").format(dateSupplier()))
+                                        .withType("ZK-Manual-Scaling")
+                                        .withReason("replicaChange")
+                                        .withStatus("True")
+                                        .withMessage(scaleMessage);
+
+                                // Setting this will add the annotation to the STS in the state.zkStatefulSet() method
+                                zkManualScaling = true;
+
+                            } else if (Annotations.booleanAnnotation(sts, Annotations.ANNO_STRIMZI_IO_MANUAL_ZK_SCALE, false)) {
+                                // This is an on-going manual scaling operation. Once the annotation has been set to true by the operator
+                                // only the user can set it to false.
+                                String scaleMessage = String.format("Detected manual Zookeeper scaling operation in progress. " +
+                                        "Rolling of Zookeeper cluster will remain disabled until %s annotation is set to false.",
+                                        Annotations.ANNO_STRIMZI_IO_MANUAL_ZK_SCALE);
+
+                                log.info(scaleMessage);
+
+                                zkManualScalingCondition = new ConditionBuilder()
```

Comment:
If I read the code right, the initial status from line 1435 will be there only until the next reocnciliation which will replace it with the next one. That sounds like very short time and possibly people might not see it. Maybe you can add some parts of that message here as well to make the status more clear (such as the reference to the docs etc.).

Reply:
Good point, I will update the status message. However, at the moment the procedure is in a markdown file in `documentation/misc`. Were we going to add it to the official docs or the release notes? Is it ok just to refer to the genrnic "docs" ?

Reply:
I think the generic reference to the docs as it is in the status above should be ok.